android/src/main/java/ee/forgr/capacitor/social/login/SocialLoginPlugin.java

```diff
@@ -54,6 +56,11 @@ public void initialize(PluginCall call) {
 
     JSObject google = call.getObject("google");
     if (google != null) {
+      if (android.os.Build.VERSION.SDK_INT < android.os.Build.VERSION_CODES.N) {
```

Comment:
this should be remove and use alternative code to make it work same version as capacitor

Reply:
I never pulled latest "development" branch. There, the problem does not happen